doc/contributing/troubleshooting.rst

```diff
@@ -27,13 +28,13 @@ See :ref:`upgrading-pypsa`.
 Consistency check on network
 ============================
 
-A consistency check can be performed using the function :py:meth:`pypsa.Network.consistency_check`.
+A consistency check can be performed using the function :py:meth:`pypsa.n.consistency_check`.
 
 
 Problems with power flow convergence
 ====================================
 
-If your ``network.pf()`` is not converging there are two possible reasons:
+If your ``n.pf()`` is not converging there are two possible reasons:
```

Comment:
Same for those, we could use references here so you can straight jump to the docstring